debian/rules

```diff
@@ -0,0 +1,7 @@
+#! /usr/bin/make -f
+
+#export DH_VERBOSE = 1
+export PYBUILD_NAME = docker-explorer
+
+%:
+	dh $@ --with python2,python3 --buildsystem=pybuild
```

Comment:
it looks like you're only using python2 so maybe remove `,python3` ?

Reply:
done